lib/tasks/list_email_templates_strings.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+def check(obj, path, result)
+  obj.each do |key, val|
+    if Hash === val
+      next_path = "#{path}#{key}"
+      if val.key?("text_body_template") && val.key?("subject_template")
+        result << next_path.sub("en.", "")
+      else
+        check(val, "#{next_path}.", result)
+      end
+    end
+  end
+end
+
+task "list_email_templates_strings" => :environment do
```

Comment:
It's worrying that we missed so many of these for years. It's nice that you made this rake task, it will make things easier for sure :ok_hand: But what do you think about extracting this out into a new integrity spec like the i18n one, that will raise an error if an email template is added without being added to `Admin::EmailTemplatesController.email_keys`?

https://github.com/discourse/discourse/blob/3541cab85648fa9cb54fa0149782d1e07ab82c59/spec/integrity/i18n_spec.rb#L32-L37

I think this will help us avoid the issue in future.